client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx

```diff
@@ -191,26 +192,37 @@ export const ChapterUsersPage: NextPageWithLayout = () => {
                       Change
                     </Button>
                   )}
-                  {is_bannable &&
-                    (bans.has(otherUser.id) ? (
-                      <Button
-                        data-cy="unbanUser"
-                        colorScheme="purple"
-                        size="xs"
-                        onClick={() => onUnban(otherUser)}
-                      >
-                        Unban
-                      </Button>
-                    ) : (
+                  {bans.has(otherUser.id) ? (
+                    <Button
+                      data-cy="unbanUser"
+                      colorScheme="purple"
+                      size="xs"
+                      onClick={() => onUnban(otherUser)}
+                    >
+                      Unban
+                    </Button>
```

Comment:
I forgot about this, but the _Unban_ action falls under the same limitations as the banning.

Reply:
If we are talking about adding `is_bannable` logic to it, should we? Because if someone is banned from chapter, they lose their role as far as I know, and instances owner can't be banned so `ban.has(owner.id)` will always be false.

Reply:
okay I will add logic for now, and when we remove the roles, we can revert it. Because owner doesn't get notification of who is unbanning who, so they may get trolled and remove every administration in rage 🤣